tee-worker/client-api/parachain-api/prepare-build/interfaces/trusted_operations/definitions.ts

```diff
@@ -37,6 +37,7 @@ export default {
                 deactivate_identity: "(LitentryIdentity, LitentryIdentity, LitentryIdentity, Option<RequestAesKey>, H256)",
                 activate_identity: "(LitentryIdentity, LitentryIdentity, LitentryIdentity, Option<RequestAesKey>, H256)",
                 request_vc: "(LitentryIdentity, LitentryIdentity, Assertion, Option<RequestAesKey>, H256)",
+                batch_request_vc: "(LitentryIdentity, LitentryIdentity, Vec<Assertion>, Option<RequestAesKey>, H256)",
```

Comment:
Order is important here. It should go after remove_identity. Same as in `tee-worker/app-libs/stf/src/trusted_call.rs`